rpm/pext.spec

```diff
@@ -0,0 +1,44 @@
+#Disable debug packages
+%define debug_package %{nil}
+
+Name:	pext
+Version:	0.9
```

Comment:
Unless RPM packages are supposed to be remade for every new release (which makes me worry about the amount of work when more package formats would be added), could this be taken from pext/VERSION or setup.py or so in some way? I always get a bit nervous when a version number is hardcoded in different places because it becomes very easy to miss changing one.

Reply:
I'll look in to this but I am not sure how feasible it is.

Reply:
Well, if not possible it's not possible and I'll just have to make a clear list of all steps to do each release and put this in here :)

Reply:
Hi!

From what I've seen in other projects they usually have a XXX.spec.in with placeholders for things like version and replace those placeholders with the correct info at ./configure or ./autogen.sh time, generating the final XXX.spec.

I'm not sure if this helps you but I thought to mention it.

Cheers!